README.md

```diff
@@ -41,6 +43,11 @@ Making use of the SORMAS web application through a mobile device web browser is
 Yes! Please download the [latest release](https://github.com/hzi-braunschweig/SORMAS-Project/releases/latest) and copy the content of /deploy/openapi/sormas-rest.yaml to an editor that generates a visual API documentation(e.g. <https://editor.swagger.io/>).
 A runtime Swagger documentation of the External Visits Resource (used by external symptom journals such as CLIMEDO or PIA) is available at ``<<host>>/sormas-rest/openapi.json`` or ``<<host>>/sormas-rest/openapi.yaml``
 
+### Sormas2Sormas
```

Comment:
Would prefer to move this into the Guidelines and Resources list below.

Reply:
fixed